cobalt/site/docs/reference/starboard/configuration-public.md

```diff
@@ -59,7 +58,6 @@ title: "Starboard Configuration Reference Guide"
 
 | Properties |
 | :--- |
-| **`SB_HAS_ON_SCREEN_KEYBOARD`**<br><br>Whether the current platform implements the on screen keyboard interface.<br><br>The default value in the Stub implementation is `0` |
```

Comment:
How does this work with `cobalt/bindings/shared/idl_conditional_macros.h` (`COBALT_ENABLE_ON_SCREEN_KEYBOARD`)? It doesn't seem very conditional

Reply:
It's very conditional, i'm backing this one out, cant remove it yet. We'll have to migrate this to Starboard extensions

Reply:
Backed it out, only bilinear filtering now